paasta_tools/setup_kubernetes_cr.py

```diff
@@ -167,9 +168,13 @@ def setup_all_custom_resources(
             continue
 
         # by convention, entries where key begins with _ are used as templates
-        raw_config_dicts = load_all_configs(
-            cluster=cluster, file_prefix=crd.file_prefix, soa_dir=soa_dir
-        )
+        raw_config_dicts = {}
+        if crd.kube_kind.singular == "flink":
+            raw_config_dicts = load_all_flink_configs(cluster=cluster, soa_dir=soa_dir)
```

Comment:
could we do this in `load_all_configs` so that the loading logic is in a single place? should be able to use `file_prefix` to determine if we need to flink-specific logic or not

Reply:
that said, i wonder if we could just have load_all_configs work like `load_service_instance_configs` and just automatically return the merged configs for everything (if not just call `load_service_instance_configs)? 

that way, future operators that want to be autotuned don't need to change any code :)

Reply:
++ 

I will work on this! Thanks for the suggestion